### ISSUE TITLE
Load My F***ing Tags errors

### ISSUE THREAD

**Issue by MaxWasUnavailable** [Ninjdai1/letsdo-addon-structures#4]
Hi there, got a couple of errors revealed through LMFT:

```
[21Aug2024 22:52:55.067] [Worker-ResourceReload-8/ERROR] [net.minecraft.tags.TagLoader/]: [Load My Fucking Tags] Couldn't load certain entries with the tag letsdoaddon-structures:has_structure/swamp_well: #bloomingnature:mangrove_swamp (from letsdo-addon-structures-1.7.1.jar), #bloomingnature:swamp (from letsdo-addon-structures-1.7.1.jar)
[21Aug2024 22:52:55.067] [Worker-ResourceReload-8/ERROR] [net.minecraft.tags.TagLoader/]: [Load My Fucking Tags] Couldn't load certain entries with the tag letsdoaddon-structures:has_structure/mangrove_outpost: #bloomingnature:mangrove_swamp (from letsdo-addon-structures-1.7.1.jar)
[21Aug2024 22:52:55.068] [Worker-ResourceReload-8/ERROR] [net.minecraft.tags.TagLoader/]: [Load My Fucking Tags] Couldn't load certain entries with the tag letsdoaddon-structures:has_structure/mangrove_hut: #bloomingnature:mangrove_swamp (from letsdo-addon-structures-1.7.1.jar)
[21Aug2024 22:52:55.070] [Worker-ResourceReload-8/ERROR] [net.minecraft.tags.TagLoader/]: [Load My Fucking Tags] Couldn't load certain entries with the tag letsdoaddon-structures:has_structure/mangrove_pond: #bloomingnature:mangrove_swamp (from letsdo-addon-structures-1.7.1.jar)
```

We're not using blooming nature, which I assume is the cause of this. Any chance this could be fixed / resolved / made to not error if a mod isn't present?

**Comment by Ninjdai1** [comment]
Already answered in #3 
TL;DR: as a datapack, if fabric and forge don't provide  (like for recipes) a way to disable a structure if a mod isn't loaded, using mod-only tags is the only way to control it
